chef/cookbooks/network/recipes/default.rb

```diff
@@ -509,4 +511,26 @@ def kill_nic(nic)
       end
     end
   end
+
+  if ovs_setup_once
+    # If we're using an ovs-bridge somewhere, enable wicked-nanny to be started
+    # for the next boot.
+    # Note: There's no need to restart wicked here as all interfaces
+    # should be correctly configure at this point.
+    template "/etc/wicked/local.conf" do
+      source "wicked-local.conf.erb"
+      owner "root"
+      group "root"
+      mode "0644"
+      variables(
+        use_nanny: true
+      )
```

Comment:
Ah, right. So the `case` will change to `case node[:platform_family]` soon (see https://github.com/crowbar/crowbar-core/pull/53), so the test should become `unless node[:platform] == "suse" && node[:platform_version] < 12.0` (to make it really SLE11-specific, and not considering openSUSE and others). Technically, just looking at the version would work, but I prefer to be slightly more explicit.